tests/navjoy_translator_test.py

```diff
@@ -0,0 +1,1065 @@
+from translator import navjoy_translator
+import xmltodict
```

Comment:
Doesn't seem to be used, can we remove? Also can we remove the commented zoneinfo below?

Reply:
removed unused imports (and sorted imports) across all translators, tools, and tests